src/components/footer/footer.js

```diff
@@ -1,27 +1,61 @@
-import { Button, Container, Grid, Link, makeStyles } from '@material-ui/core';
+import { Container, Grid, Link, makeStyles, Typography } from '@material-ui/core';
 import * as React from 'react';
 import styles from './styles';
+import StatButton from './components/StatButton';
 import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { useTotalPrizeValue } from '../../features/winners/apollo/total';
 
 const useStyles = makeStyles(styles);
 
 const Footer = ({ variant }) => {
   const { t } = useTranslation();
   const classes = useStyles();
 
+  //Holders Data
+  const cadets = useSelector(state => state.holdersReducer.totalHolders || 0);
+
+  //Buybacks Data
+  const lastWeekBuybacks = useSelector(state => state.buybacksReducer.lastBuybacksTotal || 0);
+  const potsPrice = useSelector(state => state.pricesReducer.prices['POTS'] || 0);
+  const lastWeekBuybackUSD = (lastWeekBuybacks * potsPrice).toLocaleString(undefined, {
+    maximumFractionDigits: 0,
+  });
+
+  //TVL Data
+  const tvl = useSelector(state => state.vaultReducer.totalTvl || 0).toLocaleString(undefined, {
+    maximumFractionDigits: 0,
+  });
+
+  //Total Prize Data
+  const { total } = useTotalPrizeValue();
+  const totalFormatted = total.toLocaleString(undefined, {
+    maximumFractionDigits: 0,
+  });
+
   return (
     <React.Fragment>
       <Container maxWidth="xl" className={variant === 'light' ? classes.light : ''}>
         <Grid container spacing={2} className={classes.footer}>
-          <Grid item xs={12}>
-            <Button
-              href={'https://moonpot.com/alpha'}
-              className={classes.footerBtn}
-              variant={'outlined'}
-              size={'small'}
-            >
-              {t('buttons.moonpotAlpha')}
-            </Button>
+          <Grid item xs={12} className={classes.footerLinks}>
+            <Link href={'https://docs.moonpot.com/'}>
+              <Typography className={classes.textButton}>{t('header.docs')}</Typography>
+            </Link>
+            <Link href={'https://moonpot.com/alpha'}>
+              <Typography className={classes.textButton}>{t('header.articles')}</Typography>
+            </Link>
+            <Link href={'https://vote.moonpot.com/#/'}>
+              <Typography className={classes.textButton}>{t('header.vote')}</Typography>
+            </Link>
+            <Link>
+              <Typography className={classes.textButton}>{t('header.audit')}</Typography>
+            </Link>
+          </Grid>
+          <Grid item xs={12} className={classes.footerStats}>
+            <StatButton label={t('footer.cadets')} value={cadets} />
+            <StatButton label={t('footer.prizes')} value={'$' + totalFormatted} />
+            <StatButton label={t('footer.tvl')} value={'$' + tvl} />
+            <StatButton label={t('footer.buybacks')} value={'$' + lastWeekBuybackUSD} />
```

Comment:
Not sure what the intention of the buyback stat here is (weekly/monthly/all-time?), but `lastBuyback` from the api/contract is from the last draw, not any specific time period, so adding them up doesn't make much sense.

Reply:
last buyback here is representing the buyback from the last week. I can rename the variable here if this is confusing.

Reply:
(I am summing the last buyback for each pot in the redux buybacks action)

Reply:
It is not last weeks buyback though, that would be the total of all the last draws, which could be 1 day, 1 week, 2 weeks etc.

Reply:
Thats right, my bad. I will have to work this out some other way.

Reply:
We were able to use adrians data to get a accurate value for the last weeks buyback in the last commit.